pyiron_workflow/create.py

```diff
@@ -28,11 +37,13 @@ def __init__(self):
         self.ThreadPoolExecutor = ThreadPoolExecutor
         # Local cloudpickler
         self.CloudpickleProcessPoolExecutor = CloudpickleProcessPoolExecutor
-        # executorlib
-        self.ExecutorlibExecutor = ExecutorlibExecutor
 
         self.function_node = function_node
 
+    @property
+    def executorlib(self):
+        return ExecutorlibExecutor()
```

Comment:
```suggestion
    executorlib = executorlib
```

Alternatively, we could simply directly expose the executorlib module API on the creator!

Cons: a little less tightly controlled
Pros: insanely succinct

Since @jan-janssen is following semantic versioning and exposing the key API in the top-level `__init__`, I think I lean towards this solution